packages/evolution-backend/src/services/routing/types.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
```

Comment:
We did not had routing type before when we where calling trROuting? Or that code was all in the MTL survey side ?

Reply:
Mtl called the trRouting's summary call (during the survey). For the validations, we also do direct calls to trRouting, so we get the whole response. Here, this is a new, simplified, API to get time and duration. Eventually, if/when we need it, we could add more information, if necessary. We could have another function, similar to this one, that returns the whole data. Of course, it would re-use 90% of what's here, it's just the API that would change.

Reply:
The name of the types/functions implies that we return the full results. If we are more constrainted, we need names that reflect that.

Reply:
RoutingTimeDistanceResultByMode